### PR TITLE
User can log out

### PR DESCRIPTION
**Pull request by dashlykova**
### User Story

```
As a User,
In order to exit from my account in the app,
I would like to be able to log out.
```
### Feature description
Link to this [[feature/chore in Pivotal Tracker]](https://www.pivotaltracker.com/story/show/168466794).

### Description of PR
Creates a log out component and imports it into the navbar component. Makes the logout function button/function visible after the user has logged out

### Screenshots
Screenshot 2019-09-14 at 10 11 56

### What did we learn?
Sometimes it's just best to not have a flash message

### CODE REVIEW

File: src/styling/customize.css [hunk 1]
```diff
@@ -102,6 +114,7 @@
 
 /* Modal  */
 #modal {
+  /* position: flex !important; */
```
Comment (julieahlquist) [inline src/styling/customize.css]: please take out comment

File: cypress/integration/userCanLogout.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      url: 'http://localhost:3000/api/decks',
+      response: 'fixture:flashcards.json',
+      status: 200
+    });
+    cy.user_successful_login('julie@dash.com', 'password');
+    cy.wait(2000)
+  });
+
+  it('User can successfully log out', () => {
+    debugger;
```
Comment (julieahlquist) [inline cypress/integration/userCanLogout.spec.js]: no debuggers honey

**Review by julieahlquist — CHANGES_REQUESTED** [review]

File: src/styling/customize.css (hunk 1 above)
Comment (dashlykova) [reply src/styling/customize.css]: DONE

File: cypress/integration/userCanLogout.spec.js (hunk 2 above)
Comment (dashlykova) [reply cypress/integration/userCanLogout.spec.js]: DONE

**Review by julieahlquist — APPROVED** [review]

**Review by tochman — COMMENTED** [review]
Lgtm

**Pull request merged**